src/main/java/frc/robot/commands/FiveBallCenterAuto.java

```diff
@@ -0,0 +1,91 @@
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands;
+
+import java.util.List;
+
+import edu.wpi.first.wpilibj.controller.PIDController;
+import edu.wpi.first.wpilibj.controller.RamseteController;
+import edu.wpi.first.wpilibj.controller.SimpleMotorFeedforward;
+import edu.wpi.first.wpilibj.geometry.Pose2d;
+import edu.wpi.first.wpilibj.geometry.Rotation2d;
+import edu.wpi.first.wpilibj.geometry.Translation2d;
+import edu.wpi.first.wpilibj.trajectory.Trajectory;
+import edu.wpi.first.wpilibj.trajectory.TrajectoryConfig;
+import edu.wpi.first.wpilibj.trajectory.TrajectoryGenerator;
+import edu.wpi.first.wpilibj.trajectory.constraint.DifferentialDriveVoltageConstraint;
+import edu.wpi.first.wpilibj2.command.RamseteCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.Constants;
+import frc.robot.subsystems.Carousel;
+import frc.robot.subsystems.Climber;
+import frc.robot.subsystems.DriveTrain;
+import frc.robot.subsystems.Intake;
+import frc.robot.subsystems.Shooter;
+
+public class FiveBallCenterAuto extends SequentialCommandGroup {
+  /**
+   * Add your docs here.
+   */
+  public FiveBallCenterAuto(final DriveTrain robotDrive, final Shooter shooter, final Intake intake, final Climber climber, final Carousel carousel, final DriveCommand driveCommand) {
+    driveCommand.cancel();
+    // intake.run(0.4);
+    final IntakeCommand intakeCommand = new IntakeCommand(intake, climber, Constants.INTAKE_SPEED);
+    robotDrive.resetOdometry(new Pose2d());
+    final var autoVoltageConstraint =
+        new DifferentialDriveVoltageConstraint(
+            new SimpleMotorFeedforward(Constants.ksVolts,
+                                       Constants.kvVoltSecondsPerMeter,
+                                       Constants.kaVoltSecondsSquaredPerMeter),
+            Constants.kDriveKinematics,
+            10);
+    final TrajectoryConfig config =
+      new TrajectoryConfig(Constants.kMaxSpeed,
+                            Constants.kMaxAcceleration)
+          // Add kinematics to ensure max speed is actually obeyed
+          .setKinematics(Constants.kDriveKinematics)
+          // Apply the voltage constraint
+          .addConstraint(autoVoltageConstraint);
+    final Trajectory fiveBallTrajectory = TrajectoryGenerator.generateTrajectory(
+      // Start at the origin facing the +X direction
+      new Pose2d(0, 0, new Rotation2d(0)),
+      // Pass through these two interior waypoints, making an 's' curve path
+      List.of(
+         new Translation2d(388.637 * Constants.inchToMetersConversionFactor, 173.480 * Constants.inchToMetersConversionFactor),
```

Comment:
We either need to define specific named points with these coordinates or at least put comments here so we know what they're for.